tests/providers/google/leveldb/hooks/test_leveldb.py

```diff
@@ -31,18 +31,18 @@
 
 class TestLevelDBHook:
     @mock.patch.dict("os.environ", AIRFLOW_CONN_LEVELDB_DEFAULT="test")
-    def test_get_conn_db_is_not_none(self):
+    def test_get_conn_db_is_not_none(self, tmpdir):
```

Comment:
```suggestion
    def test_get_conn_db_is_not_none(self, tmp_path):
```

`tmpdir` fixture use legacy `py.path.local` the [preferable](https://docs.pytest.org/en/stable/how-to/tmp_path.html#the-tmpdir-and-tmpdir-factory-fixtures) fixture `tmp_path` which use pathlib

Reply:
RIght.. Good point :)

Reply:
Fixed,